README.md

```diff
@@ -1,2 +1,54 @@
 # MyWeb
-for project: Project_myweb
+for project: Project_myweb/MyWeb
+
+### Requirements:
+
+Version:
+- Python3: 3.10.12
+- Nginx: nginx/1.18.0 (Ubuntu)
+- Flask: 3.0.3
+- Werkzeug 3.0.4
+- Ubuntu: 22.04
+- Docker: 27.3.1, build ce12230
+- Docker-compose: 2.28.1
+- Ansible: core 2.16.11
+- jinja: 3.1.4
+
+# Build image:
+1. Clone the repo
+```bash
+git clone https://github.com/matusik-ops/MyWeb
+```
+2. Build it
+```bash
+cd ./MyWeb
+docker build -f Docker/Dockerfile -t testimg . # docker build -f <path/to/Dockerfile> -t <image_name> . 
+```
+
+# How to run it
+
+1. Install update
```

Comment:
What does it mean? IMO it could be removed

Reply:
What do you think?

Reply:
Can you clarify what exactly you mean by "Install update"? Could you explain what you are expecting in that context?

Reply:
It would be better. Check it.

Reply:
all clear now